doc/Language/compilation.pod6

```diff
@@ -0,0 +1,291 @@
+=begin pod :kind("Language") :subkind("Language") :category("tutorial")
+
+=TITLE Compilation
```

Comment:
Maybe "Precompilation" would be a better title? Or CompUnits?

Reply:
Since this is a tutorial, rather than reference, I thought that 'Compilation' would be something most developers would have some idea about. 'Precompilation' is something that seems to me (from my reading) to be more Raku related.
Even more so 'CompUnit'

Reply:
But we're not really talking about compilation, but about CompUnits and how to work with them. Even if it's Raku related, it's what we're talking about here.